### ISSUE TITLE
Allowing option to set the destination folder for the created image

### ISSUE THREAD

**Issue by rohan-paul** [kufii/CodeSnap#41]
Currently the image is being created in the same directory of the file into which I am pasting the image from clipboard.

But it would be great if we can set a directory, e.g. /assets/ where the created image will be pasted.

**Comment by kufii** [comment]
if you click the snap button a file selector should pop up. Do you mean you want it to auto save without prompting? If so that is a feature that can be added

**Comment by rohan-paul** [comment]
Yes, I meant for auto save without promting. Or even if prompting it remembers my last folder selection and the prompt shows that last folder in the prompt next time. For example, while pasting in a markdown, I keep all the screenshort in a sepearte folder named 'assets'. So now, I have to type that 'assets/' after I paste a screenshot to a markdown file for example.

Its very helpful when pasting large number of screenshot into a markdown document e.g. while writing tutorial or documentation.

**Comment by kojoYeboah53i** [comment]
dude, I can't find the directory where the snapshot is saved into. I'm using windows 10. Am I the only one having this problem?
